exporter/index.js

```diff
@@ -0,0 +1,148 @@
+const express = require('express');
+const puppeteer = require('puppeteer');
+const validate = require('express-validation');
+const Joi = require('joi');
+const Raven = require('raven');
+const bodyParser = require('body-parser')
+const cors = require('cors')
+
+const validation = {
+  screenshot: {
+    body: {
+      format: Joi.string().valid(['png', 'pdf']),
+      target: Joi.string().uri().required(),
+      title: Joi.string(),
+      selector: Joi.string().required(),
+    },
+  },
+};
+
+if (process.env.SENTRY_DSN) {
+  Raven.config(process.env.SENTRY_DSN).install();
+}
+
+const captureException = error => {
+  if (process.env.SENTRY_DSN) {
+    Raven.captureException(error);
+  } else {
+    console.error(error);
+  }
+};
+
+const setContext = (contextData, callback) => {
+  if (process.env.SENTRY_DSN) {
+    Raven.context(function () {
+      Raven.setContext(contextData);
+      callback();
+    });
+  } else {
+    callback();
+  }
+};
+
+const app = express();
+let browser;
+let context;
+
+app.use(bodyParser.json());
+app.use(cors());
+
+(async() => {
+  browser = await puppeteer.launch({
+    args: [
+      '--no-sandbox',
+      '--disable-setuid-sandbox',
+    ],
+  });
+  // Create a new incognito browser context.
+  context = await browser.createIncognitoBrowserContext();
```

Comment:
Since we send auth token to the exporter don' think we should reuse the browser context, I think we should create a new context per user request. Reading the docs on browser.createIncognitoBrowserContext() "Creates a new incognito browser context. This won't share cookies/cache with other browser contexts." https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#browsercreateincognitobrowsercontext